README.md

```diff
@@ -16,7 +16,8 @@ Existing resource group
 ```hcl
 module "resource_group" {
   # Replace "main" with a GIT release version to lock into a specific release
-  source                       = "git::https://github.com/terraform-ibm-resource-group/terraform-ibm-resource-group.git?ref=main"
```

Comment:
Please delete the main comment line 18 (legacy, no longer needed). The replacement comment is good.